packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -547,79 +519,35 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
     totalItemCount: number,
     hasCheckmarks?: boolean,
     hasIcons?: boolean) {
+    const { contextualMenuItemAs } = this.props;
+    const dismissMenu = this.dismiss.bind(this, undefined);
     const { expandedMenuItemKey } = this.state;
-    const { contextualMenuItemAs: ChildrenRenderer = ContextualMenuItem } = this.props;
-
-    const subMenuId = this._getSubMenuId(item);
-    let ariaLabel = '';
-
-    if (item.ariaLabel) {
-      ariaLabel = item.ariaLabel;
-    } else if (item.name) {
-      ariaLabel = item.name;
-    }
-
-    const isChecked: boolean | null | undefined = getIsChecked(item);
-    const canCheck: boolean = isChecked !== null;
-    const defaultRole = canCheck ? 'menuitemcheckbox' : 'menuitem';
-    const itemHasSubmenu = hasSubmenu(item);
-
-    const buttonNativeProperties = getNativeProps(item, buttonProperties);
-    // Do not add the disabled attribute to the button so that it is focusable
-    delete (buttonNativeProperties as any).disabled;
-
-    const itemButtonProperties = {
-      className: classNames.root,
-      onClick: this._onItemClick.bind(this, item),
-      onKeyDown: itemHasSubmenu ? this._onItemKeyDown.bind(this, item) : null,
-      onMouseEnter: this._onItemMouseEnter.bind(this, item),
-      onMouseLeave: this._onMouseItemLeave.bind(this, item),
-      onMouseDown: (ev: any) => this._onItemMouseDown(item, ev),
-      onMouseMove: this._onItemMouseMove.bind(this, item),
-      href: item.href,
-      title: item.title,
-      'aria-label': ariaLabel,
-      'aria-haspopup': itemHasSubmenu || undefined,
-      'aria-owns': item.key === expandedMenuItemKey ? subMenuId : undefined,
-      'aria-expanded': itemHasSubmenu ? item.key === expandedMenuItemKey : undefined,
-      'aria-checked': !!isChecked,
-      'aria-posinset': focusableElementIndex + 1,
-      'aria-setsize': totalItemCount,
-      'aria-disabled': isItemDisabled(item),
-      role: item.role || defaultRole,
-      style: item.style
-    };
-
-    let { keytipProps } = item;
-    if (keytipProps && itemHasSubmenu) {
```

Comment:
This was not properly copied over, will fix

Reply:
Have you fixed this?